ServerModules.cfg

```diff
@@ -1,6 +1,7 @@
 # Comment out modules using a `#` to exclude
 # a module from a build
 
+ApplyMsgFunctions
```

Comment:
While I see at least one exception, could you rename this as `ApplyMsg` for better consistency? `ApplyMsgFunctions` sounds like internal helpers to me (Sparse support has a similar module structure IIRC)